.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "dev environment",
+	"dockerFile": "Dockerfile",
+	"extensions": [
+		"vscjava.vscode-java-pack",
+		"scala-lang.scala",
+		"scalameta.metals",
+		"scala-lang.scala-snippets",
+		"eamodio.gitlens",
```

Comment:
This extension does not seem relevant to the change, please remove it
```suggestion
```

Reply:
done